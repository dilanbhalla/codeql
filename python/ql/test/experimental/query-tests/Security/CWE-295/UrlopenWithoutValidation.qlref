experimental/Security/CWE-295/UrlopenWithoutValidation.ql