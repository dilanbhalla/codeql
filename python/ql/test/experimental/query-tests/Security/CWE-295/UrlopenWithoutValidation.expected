| urlopen.py:4:1:4:62 | ControlFlowNode for Attribute() | This call to urlopen does not provide any certificate validation |
| urlopen.py:6:1:6:44 | ControlFlowNode for Attribute() | This call to urlopen does not provide any certificate validation |
| urlopen.py:7:1:7:57 | ControlFlowNode for Attribute() | This call to urlopen does not provide any certificate validation |
| urlopen.py:12:1:12:42 | ControlFlowNode for urlopen() | This call to urlopen does not provide any certificate validation |
| urlopen.py:18:5:18:46 | ControlFlowNode for openurl() | This call to urlopen does not provide any certificate validation |
| urlopen.py:23:5:23:48 | ControlFlowNode for openurl() | This call to urlopen does not provide any certificate validation |
